plugins/httpapi/ftd.py

```diff
@@ -22,7 +22,7 @@
 
 DOCUMENTATION = '''
 ---
-author: Ansible Networking Team
+author: Ansible Networking Team (!UNKNOWN)
```

Comment:
```suggestion
author: Ansible Networking Team (!UNKNOWN)
```
is it needed here?

Reply:
Yes, since `Ansible Networking Team` is not a valid value (see https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py#L25-L32 for the valid values).

Reply:
ok than, approved below